subiquity/ui/views/filesystem/guided.py

```diff
@@ -98,13 +111,72 @@ def cancel(self, btn=None):
 
 A LVM volume group is created containing the large partition. A \
 4 gigabyte logical volume is created for the root filesystem. \
+It can easily be enlarged with standard LVM command line tools."""),
+    'luks': _("""The LUKS encrypted LVM guided partitioning scheme creates three \
+partitions on the selected disk: one as required by the bootloader, one \
+for '/boot', and one covering the rest of the disk.
+
+A LUKS encrypted LVM volume group is created containing the large partition. A \
+4 gigabyte logical volume is created for the root filesystem. \
 It can easily be enlarged with standard LVM command line tools."""),
 }
 
 
 def _wrap_button_row(row):
     return CursorOverride(Color.done_button(row), 2)
 
+loss_warning = _("""Warning: If you lose this security key, all data will be lost. If \
+you need to, write down your key and keep it in a safe place \
+elsewhere.""")
+
+
+class GuidedPasswordForm(Form):
+
+    def __init__(self):
+        super().__init__()
+        setup_password_validation(self, _("Security key"))
+
+    cancel_label = _("Back")
+    password = PasswordField(_("Choose a security key:"))
+    confirm_password = PasswordField(_("Confirm the security key:"),
+                                     help=loss_warning)
+
+    def validate_password(self):
+        if len(self.password.value) < 1:
+            return _("Security key must be set")
+
+    def validate_confirm_passwor(self):
+        if self.password.value != self.confirm_password.value:
+            return _("Security keys do not match")
+
+class GuidedPasswordView(BaseView):
+    title = _("Choose a security key")
+    footer = _("")
+    excerpt = _("""Disk encryption protects your files in case you lose your \
+computer. It requires you to enter a security key each time the \
```

Comment:
Wrapping seems odd here.

Reply:
```
>>> print("""long string \
... without leading chars.
... 
... and again long string \
...      but with leading indentation.
... """)
long string without leading chars.

and again long string      but with leading indentation.

```

Reply:
i'm not sure how to wrap things, without introducing additional whitespace, whilst also wrapping to a reasonable source code line width.